cdap-examples/Purchase/src/main/java/co/cask/cdap/examples/purchase/PurchaseApp.java

```diff
@@ -59,6 +61,11 @@ public void configure() {
     // Provide a Service to Application components
     addService(new CatalogLookupService());
 
+    // Add schedule to the Application which can be referred by underlying programs
```

Comment:
Add a Schedule to the Application which can be referred to by other programs such as a Workflow